src/lib/nfs-workarounds.c

```diff
@@ -146,16 +146,23 @@ int nfs_safe_link(const char *oldpath, const char *newpath, bool links1)
 
 static void nfs_flush_chown_uid(const char *path)
 {
-	uid_t uid;
-
 #ifdef ATTRCACHE_FLUSH_CHOWN_UID_1
-	uid = (uid_t)-1;
+    if (chown(path, (uid_t)-1, (gid_t)-1) < 0) {
+        if (errno == ESTALE || errno == EPERM) {
+            /* attr cache is flushed */
+            return;
+        }
+        if (likely(errno == ENOENT)) {
```

Comment:
`} else if (`

Reply:
Thank you for the review, I have adjusted the if-statements inside the function to use `} else if (`